nengo_loihi/block.py

```diff
@@ -124,32 +127,32 @@ class Compartment:
         Compartment voltage thresholds.
     """
     # threshold at which U/V scaling is allowed
-    DECAY_SCALE_TH = 0.5 / 2**12  # half of one decay scaling unit
+    DECAY_SCALE_TH = 0.5 / d(b'NDA5Ng==', int)  # half of decay scaling unit
 
     def __init__(self, n_compartments, label=None):
         self.n_compartments = n_compartments
         self.label = label
 
         # parameters specific to compartments/block
-        self.decayU = np.ones(n_compartments, dtype=np.float32)
+        self.decay_u = np.ones(n_compartments, dtype=np.float32)
         # ^ default to no filter
-        self.decayV = np.zeros(n_compartments, dtype=np.float32)
+        self.decay_v = np.zeros(n_compartments, dtype=np.float32)
         # ^ default to integration
         self.tau_s = None
-        self.scaleU = True
-        self.scaleV = False
+        self.scale_u = True
+        self.scale_v = False
 
-        self.refractDelay = np.zeros(n_compartments, dtype=np.int32)
+        self.refract_delay = np.zeros(n_compartments, dtype=np.int32)
         self.vth = np.zeros(n_compartments, dtype=np.float32)
         self.bias = np.zeros(n_compartments, dtype=np.float32)
-        self.enableNoise = np.zeros(n_compartments, dtype=bool)
+        self.enable_noise = np.zeros(n_compartments, dtype=bool)
 
         # parameters common to core
         self.vmin = 0
         self.vmax = np.inf
-        self.noiseMantOffset0 = 0
-        self.noiseExp0 = 0
-        self.noiseAtDendOrVm = 0
+        self.noise_offset = 0
+        self.noise_exp = 0
+        self.noise_dend = 0
```

Comment:
I'm pretty sure that a 0 here indicates "dendrites", and a 1 indicates "membrane". So maybe call this "noise_at_membrane" or something like that? Also, it should be documented above (I guess we just forgot before).